src/Pagination/components/Pages.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTokens from "../../defaultTokens";
+import type { Props } from "./Pages";
+import Button from "../../Button";
+import PageButtonLink from "./PageButtonLink";
+
+const StyledButton = styled.div`
+  &:hover,
+  &:active,
+  &:focus {
+    background: ${({ theme }) => theme.orbit.backgroundButtonSecondary};
+    color: ${({ theme }) => theme.orbit.colorTextButtonSecondary};
+    transform: none;
+    cursor: default;
+  }
+`;
+
+StyledButton.defaultProps = {
+  theme: defaultTokens,
+};
+
+const ActiveButton = ({ children }) => (
+  <Button type="secondary" component={props => <StyledButton {...props} type={undefined} />}>
+    {children}
+  </Button>
+);
+
+const Pages = ({ pageCount, selectedPage, onPageChange, enlargement = 1 }: Props) =>
+  // $FlowFixMe
+  Array(...Array(pageCount)).map((_, index) => {
```

Comment:
Probably u can write something like:

```
const iterate: React$Node = Array(...Array(pageCount)).map((_, index) => {
    const key = index + enlargement;
    return selectedPage === key ? (
      <ActiveButton type="secondary">{key}</ActiveButton>
    ) : (
      <PageButtonLink onPageChange={onPageChange}>{key}</PageButtonLink>
    );
  });
  return iterate;
```
there is no flow error in this case. Not sure about it, just offer :)